src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+import { Error } from '../../types/TypeOfErrors';
+
+interface Props {
+  error: Error | null,
+  setError: (value: Error | null) => void;
+}
+
+export const Errors: React.FC<Props> = ({
+  error,
+  setError,
+}) => {
+  useEffect(() => {
+    setTimeout(() => {
+      setError(null);
+    }, 3000);
+  }, [error, setError]);
```

Comment:
you need to add state into array of deps, not a function, `setError` should not be here